src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentBindLoweringPass.cs

```diff
@@ -207,67 +210,28 @@ private IntermediateNode[] RewriteUsage(IntermediateNode parent, TagHelperProper
                 format = GetAttributeContent(formatNode);
             }
 
-            // Now rewrite the content of the value node to look like:
-            //
-            // BindMethods.GetValue(<code>) OR
-            // BindMethods.GetValue(<code>, <format>)
             var valueExpressionTokens = new List<IntermediateToken>();
-            valueExpressionTokens.Add(new IntermediateToken()
-            {
-                Content = $"{ComponentsApi.BindMethods.GetValue}(",
-                Kind = TokenKind.CSharp
-            });
-            valueExpressionTokens.Add(original);
-            if (!string.IsNullOrEmpty(format?.Content))
+            var changeExpressionTokens = new List<IntermediateToken>();
+            if (changeAttribute == null || changeAttribute.IsDelegateProperty())
```

Comment:
In this commit, I'm only updating `bind-...` for components that have an `EventCallback<T>`` property as their change handler. DOM-based bind still goes down the delegate path (see next commit)